themes/mongodb/src/css/code-buttons.css

```diff
@@ -84,3 +88,42 @@
         display: none;
     }
 }
+
+
+/* linenos code-block styling */
+table.highlighttable {
+  table-layout: fixed;
+}
+
+table.highlighttable td {
+  padding: 0;
+}
+
+td.linenos {
+  padding-top: 0;
+}
+
+td.linenos pre {
+  padding-top: 0;
+}
+
+td.code {
+  width: 100%;
+  padding-top: 0;
+}
+
+td.code div.highlight {
+  padding-top: 0;
+}
+
+td.code div.highlight pre {
+  padding-top: 0;
+}
+
+td.code pre {
+    max-width: 100%;
+}
+
+#linenos-button-row-filler {
```

Comment:
Should this be a class rather than an id? That is, can't there be multiple of these instances? Duplicate ids are invalid.